data/SohamG934.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "Soham Ghadge",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "**Budding Engineer** and **tech aficionado**. **Travel enthusiast**",
+	"links": [{
+		"name": "Follow me on Twitter",
+		"url": "https://twitter.com/SOHAMG5",
+		"icon": "FaTwitter"
+	}, 
+            {
+		"name": "Follow me on LinkedIn",
+		"url": "https://www.linkedin.com/in/soham-ghadge-17bb18245/",
+		"icon": "FaLinkedIn"
```

Comment:
```suggestion
		"icon": "FaLinkedin"
```

Reply:
Thank you for pointing out the error, it was overlooked by me.